library_generation/generate_repo.py

```diff
@@ -102,6 +108,11 @@ def generate_from_yaml(
             versions_file=repo_config.versions_file,
         )
 
+    # we skip repo_level_postprocess if not in a monorepo since it's meant for
```

Comment:
I know it was not introduced in this PR, can we rename `repo_level_postprocess` to `monorepo_postprocess` if it's meant for monorepo only?

Reply:
Agreed. Renamed to `monorepo_postprocessing`